js/views/itemShortVw.js

```diff
@@ -13,14 +13,16 @@ module.exports = Backbone.View.extend({
     'click .js-itemShortEdit': 'editItemClick',
     'click .js-itemShortDelete': 'deleteItemClick',
     'click .js-userShortFollow': 'followUser',
-    'click .js-userShortUnfollow': 'unfollowUser'
+    'click .js-userShortUnfollow': 'unfollowUser',
+    'click .js-blockUser': 'blockUser',
+    'click .js-unblockUser': 'unblockUser'
   },
 
   initialize: function(options){
     //pre-load image
     var self=this;
     this.parentEl = $(options.parentEl);
-    this.listenTo(this.model, 'change:priceSet', this.render);
+    this.listenTo(this.model, 'change', this.render);
```

Comment:
I'm not seeing a double render happening, so it's probably a moot point for now.

I don't have a complete understanding of what the idea is, but if there were double renders, I think it's the first one (before the attrs are updated) that we'd want to eliminate. Probably, moving this to parse or sync would be the right way to go.